README.md

```diff
@@ -1,2 +1,21 @@
 # semester_4_python
 Repository for homeworks in 4th semester in SPbSU programming course
+### Homework №3
+1. **Анализ таблицы**:
+    1. Визуализировать базовую статистику таблицы. В каком классе было больше всего пассажиров?
+    2. Группировать таблицу в два уровня: класс и пол, по среднему значению возраста. Кто из возможных комбинаций самый юный, кто самый взрослый? Насколько отличаются эти значения?
+    3. Отобрать только выживших пассажиров с фамилией, начинающейся на “K”. Отсортировать их по убыванию стоимости билета. Кто заплатил больше всех? Кто меньше всех?
+    4. Какое максимальное количество родных было с выжившим пассажиром?
+    5. Посчитайте среднюю стоимость билета пассажиров, для которых указана каюта (Cabin) и для тех, у кого она не указана, во сколько раз они отличаются?
+
+2. **Визуализация**: необходимо построить по данным таблицам 10 различных графиков. Использовать можно любые средства, однако рекомендуются Seaborn и Plotly. 5 графиков должны быть интерактивными (Plotly), 1 график необходимо сделать в 3D. С некоторыми типами графиков предстоит разобраться самим, среди представленных должны быть:
+	1. Scatter plot;
+	2. Linear plot (with several lines);
+	3. Histogram;
```

Comment:
This is also not suitable: your "hsitogram" actually turned out to be a bar chart. Historgram should be used for continuous data, for numerical values (like age, for instance).